index.html

```diff
@@ -38,146 +38,138 @@
     <meta name="theme-color" content="#ffffff">
     <title>App, Website and Software Development & Training| London | dwyl</title>
   </head>
-  <body>
+  <body class="dwyl-bg-dark-gray">
     <!-- Google Tag Manager (noscript) -->
     <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WSDB8SX"
     height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
     <!-- End Google Tag Manager (noscript) -->
-    <nav class="w-100 dwyl-bg-dark-gray tc white relative fixed z-1">
+    <div class="w-100 dwyl-bg-dark-gray h3 tc white fixed shadow-5 z-1"></div>
+    <div class="w-100 dwyl-bg-dark-gray h3 mt2 pb3 absolute z-2"></div>
+    <nav class="w-100 tc white fixed bg-transparent-l dwyl-bg-dark-gray z-3">
```

Comment:
Agreed that the colour here needs to change (the two dark greys next to each other are weird), but the drop-shadow needs to stay (I think it was in the `ul` before):
https://github.com/dwyl/dwyl-site/issues/407#issuecomment-334308449